### ISSUE TITLE
<script class="__REQUESTLY__SCRIPT">... added to response

### ISSUE THREAD

**Issue by pszemus** [requestly/customer-support#65]
Hi
My Requestly Chrome plugin is configured to modify headers in HTTP response:

- Modify -> Response -> Header: Content-Type -> Value: text/xml
- Request: Path -> Contains: .mpd

It worked well for quite some time but recently I've noticed that responses for .mpd files have added
```
<script class="__REQUESTLY__SCRIPT">(function(namespace) { window[namespace] = window[namespace] || {}; window[namespace].responseRules = {}; let open = XMLHttpRequest.prototype.open; XMLHttpRequest.prototype.open = function(method) { this.addEventListener('readystatechange', function() { if (this.readyState === 4 && window[namespace].responseRules.hasOwnProperty(this.responseURL)) { const responseRule = window[namespace].responseRules[this.responseURL]; const {response, id} = responseRule; const responseType = this.responseType; let customResponse; customResponse = response.type === 'code' ? responseRule.evaluator({ method, url: this.responseURL, requestHeaders: this.requestHeaders, requestData: this.requestData, responseType: this.responseType, response: this.response }) : response.value; Object.defineProperty(this, 'response', { get: function () { if (response.type === 'static' && responseType === 'json') { return JSON.parse(customResponse); } return customResponse; } }); if (responseType === '' || responseType === 'text') { Object.defineProperty(this, 'responseText', { get: function () { return customResponse; } }); } window.postMessage({ from: 'requestly', type: 'response_rule_applied', id }, window.location.href); } }, false); open.apply(this, arguments); }; let send = XMLHttpRequest.prototype.send; XMLHttpRequest.prototype.send = function(data) { this.requestData = data; send.apply(this, arguments); }; let setRequestHeader = XMLHttpRequest.prototype.setRequestHeader; XMLHttpRequest.prototype.setRequestHeader = function(header, value) { this.requestHeaders = this.requestHeaders || {}; this.requestHeaders[header] = value; setRequestHeader.apply(this, arguments); } })('__REQUESTLY__')</script>
```
in their body.

Sample URL: https://storage.googleapis.com/shaka-live-assets/player-source.mpd
and response:
![image](https://user-images.githubusercontent.com/13220781/84748094-75000400-afb8-11ea-841f-ce9642c3c41f.png)

**Comment by requestly-team** [comment]
Hey @pszemus, Thanks for logging this issue. This is supposed to be inserted in the documents (Web pages) in order to support a new feature we have introduced recently to modify the response of AJAX requests

Is this breaking something for you? We will look into this on an urgent basis.

**Comment by pszemus** [comment]
I'm using Requestly to view contents of .mpd and .m3u8 files inside my browser. For that I need to modify response's Content-Type header as described in this issue.
That inserted <script> does not break anything except that there's an extra content inside response that I'm viewing.

**Comment by requestly-team** [comment]
@pszemus We have identified and fixed the issue. Since this is not blocking anything, we are not rolling out a hotfix. Ret assured, this will be available soon with our next release in Chrome Store. Thanks once again for the details.

**Comment by requestly-team** [comment]
Hey @pszemus, We have released v20.9.9 and can you please check if it is still happening in the latest version.

**Comment by pszemus** [comment]
Now everything looks right! Thank you.
